src/lib/journey/stages/webauthn.svelte

```diff
@@ -114,15 +128,26 @@
       </p>
     </header>
   {:else}
-    <header id={formHeaderId}>
+    <header class={`tw_input-spacing`} id={formHeaderId}>
+      {#if !askForDeviceName}
```

Comment:
This too. If `askForDeviceName` is false, we show the input? I feel like these are opposite to what we actually want.